docs/development/reviewing-patches.rst

```diff
@@ -33,6 +33,32 @@ These are small things that are not caught by the automated style checkers.
 * Does a variable need a better name?
 * Should this be a keyword argument?
 
+Testing with twine
+------------------
+If you are testing Warehouse locally, you may want to use `twine`_ to try
+uploading a test package. First, checkout the branch you would like to test.
+Then, start up the development environment (as described in :ref:`getting-started`).
+Once you have the Warehouse site working on ``localhost:80``, you can upload a
+package to the version of Warehouse running in your development environment
+with Twine using the following command:
+
+.. code-block:: console
+
+  twine upload --repository-url http://localhost/legacy/ -u username -p password path to distribution
+
+In the command above,
+
+-   ``<path to distribution>`` should be a path to a distribution file (or
+    directory of distribution files) of the package you would like to test,
+    likely created by running ``python setup.py``.
+
+-   ``<username>`` is the username of the account you would like to test with.
+    Note that your current PyPI username may not exist in the development
+    environment.
+
+-   ``<password>`` is the literal string "password". All passwords in the
+    development environment have been set to this value.
```

Comment:
`password` shouldn't be in brackets here because it is the actual value `password` and isn't something that should be replaced.